src/v2/Apps/Order/Routes/__tests__/Payment.jest.tsx

```diff
@@ -208,23 +234,34 @@ describe("Payment", () => {
     })
 
     it("renders selection of payment methods", async () => {
-      const page = await buildPage()
-      expect(page.text()).toContain("Credit Card")
-      expect(page.text()).toContain("Bank Transfer")
+      const env = setupTestEnv()
+      const page = await env.buildPage()
+      expect(page.text()).toContain("Credit card")
+      expect(page.text()).toContain("Bank transfer")
     })
 
     it("renders credit card element when credit card is chosen as payment method", async () => {
-      const page = await buildPage()
-      page.selectPaymentMethod(0)
-      expect(page.text()).toContain("Credit card")
-      expect(page.find(PaymentPickerFragmentContainer).length).toBe(1)
+      const env = setupTestEnv()
+      const page = await env.buildPage()
+      page.selectPaymentMethod(1)
+      const creditCardCollapse = page.find(PaymentPickerFragmentContainer).closest(Collapse)
+      expect(creditCardCollapse.first().props().open).toBe(true)
+      const bankDebitCollapse = page.find(BankDebitProvider).closest(Collapse)
+      expect(bankDebitCollapse.first().props().open).toBe(false)
     })
 
-    it("renders bank element when bank transfer is chosen as payment mehod", async () => {
-      const page = await buildPage()
-      page.selectPaymentMethod(1)
-      expect(page.text()).toContain("Bank transfer")
-      expect(page.find(BankDebitProvider).length).toBe(1)
```

Comment:
Checking for presence doesn't work in this case because we are using `Collapse` and it's just changing the element heights. Updated below to check the props on the corresponding `Collapse` component.